### PR TITLE
Alejandra

### PR DESCRIPTION
**Pull request by alejanmg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # git-demo
 to demonstrate how to use git and GitHub
+alejandra munoz garcia
```
Comment (beanumber) [inline README.md:3]: Could you please add a `-` so this becomes a list item?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+https://github.com/alejanmg
```
Comment (beanumber) [inline README.md:4]: Also, please make your name into a link that goes to that URL. The URL should not be visible.

**Review by beanumber — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (beanumber) [reply README.md:4]: @alejanmg have you had a chance to look at this again?

**Review by mariumtapal — CHANGES_REQUESTED** [review]
The dash needs to be outside the square bracket with a space between the two in order to make it a list item! Could you make this change?

**Review by mariumtapal — APPROVED** [review]

**Pull request merged**